### ISSUE TITLE
Use Bluebird to enforce async/await

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#140]
### Problem

The localization tests are unwieldy and don't conform to our [updated contribution reqs](https://github.com/okTurtles/group-income-simple/blob/master/CONTRIBUTING.md#pull-request-requirements) which enforce async/await semantics (for better readability & simplicity) over callback-hell semantics.

### Solution

Use Bluebird to [`promisifyAll`](http://bluebirdjs.com/docs/api/promise.promisifyall.html) the `fs` module so that this (and surrounding code):

```js
            fs.unlink(path, (err) => {
              if (err) {
                throw err
              }
              fs.unlink(output, (err) => {
                if (err) {
                  throw err
                }
                done()
              })
            })
```

Turns into this:

```js
            await fs.unlinkAsync(path)
            await fs.unlinkAsync(output)
```

Mocha has [native support for promises](https://mochajs.org/#working-with-promises) in tests. It requires not using the `done` callback. The other tests show how that works.

**Comment by vijayee** [comment]
corrected in #155

**Comment by taoeffect** [comment]
Nice, and you did it the sane way with the existing `unlinkSync` APIs 👍
